spring-cloud-task-stream/src/main/java/org/springframework/cloud/task/launcher/TaskLauncherConfiguration.java

```diff
@@ -44,4 +52,25 @@ public TaskLauncher taskLauncher() {
 		}
 	}
 
+	@Bean
+	public MavenResourceLoader mavenResourceLoader(MavenProperties properties) {
+		return new MavenResourceLoader(properties);
+	}
+
+	@Bean
+	@ConditionalOnMissingBean(DelegatingResourceLoader.class)
+	public DelegatingResourceLoader delegatingResourceLoader(MavenResourceLoader mavenResourceLoader) {
+		Map<String, ResourceLoader> loaders = new HashMap<>();
+		loaders.put("maven", mavenResourceLoader);
+		return new DelegatingResourceLoader(loaders);
+	}
+
+	@Bean
+	public MavenProperties mavenProperties() {
+		return new MavenConfigurationProperties();
+	}
+
+	@ConfigurationProperties(prefix = "maven")
+	static class MavenConfigurationProperties extends MavenProperties {
+	}
```

Comment:
Isn't all of this really dependent on the launcher?  AKA if we swapped the K8s launcher for the local task launcher, we wouldn't need this maven stuff, we'd need docker stuff, right?